administrator/components/com_cookiemanager/access.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<access component="com_cookiemanager">
+	<section name="component">
+		<action name="core.admin" title="JACTION_ADMIN" />
+		<action name="core.options" title="JACTION_OPTIONS" />
+		<action name="core.manage" title="JACTION_MANAGE" />
+		<action name="core.create" title="JACTION_CREATE" />
+		<action name="core.delete" title="JACTION_DELETE" />
+		<action name="core.edit" title="JACTION_EDIT" />
+		<action name="core.edit.state" title="JACTION_EDITSTATE" />
+		<action name="core.edit.own" title="JACTION_EDITOWN" />
+		<action name="core.edit.value" title="JACTION_EDITVALUE" />
+		<action name="core.execute.transition" title="JACTION_EXECUTETRANSITION" />
```

Comment:
is this action possible or needed in cookie management

Reply:
First I will read docs and then I will update it, but create and delete not needed but can edit is needed 
Here I don't know about Configure ACL & Options, Configure Options Only etc.